src/Tracing/TransactionContext.php

```diff
@@ -65,4 +66,34 @@ public function setParentSampled(?bool $parentSampled): void
     {
         $this->parentSampled = $parentSampled;
     }
+
+    /**
+     * Returns a context populated with the data of the given header.
+     *
+     * @param string $header The sentry-trace header from the request
+     *
+     * @return static
+     */
+    public static function fromSentryTrace(string $header)
+    {
+        $context = new static();
```

Comment:
Since the class is `final` we can now transform this to `new self()` and also add the typehint to the method rather than using the annotation